src/Sarif.Converters/FortifyFprConverter.cs

```diff
@@ -38,7 +38,7 @@ internal class FortifyFprConverter : ToolFileConverterBase
         private readonly ToolComponent CweToolComponent = new ToolComponent
         {
             Name = "CWE",
-            Guid = "2B841697-D0DE-45DD-9F19-1EEE1312429",
+            Guid = "25F72D7E-8A92-459D-AD67-64853F788765",
```

Comment:
>25F72D7E-8A92-459D-AD67-64853F788765 [](start = 20, length = 36)

What is the source of this guid and how did we know that it changed? (perhaps we should add a comment in the code well)

Reply:
if yes, why?

---
In reply to: [313676882](https://github.com/microsoft/sarif-sdk/pull/1641#discussion_r313676882) [](ancestors = 313676882)

Reply:
Oh, looked at the sarif files to understand - it appears this is probably because we introduce it as a key to refer to in places like supportedTaxonomies --> taxa. BUT, doesnt index serve the same purpose?

---
In reply to: [313677253](https://github.com/microsoft/sarif-sdk/pull/1641#discussion_r313677253) [](ancestors = 313677253,313676882)

Reply:
ok, talking to myself here, but according to spec - if only index is present in a toolComponentReference, it is assumed to be referencing tool.extensions only. So, we do need a guid.

leaving this whole thing up to validate my understanding :)

---
In reply to: [313677918](https://github.com/microsoft/sarif-sdk/pull/1641#discussion_r313677918) [](ancestors = 313677918,313677253,313676882)